src/labels/CustomLabels.labels

```diff
@@ -2158,6 +2158,14 @@
         <protected>true</protected>
         <shortDescription>Validate Recurring Donations data if it is valid install period</shortDescription>
         <value>{0} is not a supported value for Installment Period. Select Daily, Weekly, 1st and 15th, Monthly, or Yearly.</value>
+    </labels>
+     <labels>
+        <fullName>RD2_InstallmentStatusSkipped</fullName>
```

Comment:
@sam-knox , can you please verify content for seven new custom labels?

Reply:
@sam-knox , here are screenshots containing labels:
<img width="702" alt="Screen Shot 2020-07-20 at 5 58 37 PM" src="https://user-images.githubusercontent.com/28743286/88001499-23fe9680-cab5-11ea-8e0a-fcd1fbe479f2.png">

<img width="706" alt="Screen Shot 2020-07-20 at 5 56 10 PM" src="https://user-images.githubusercontent.com/28743286/88001511-2bbe3b00-cab5-11ea-8ca1-5d39e998f502.png">

Upcoming Installments component displaying skipped Installments:
<img width="511" alt="Screen Shot 2020-07-20 at 5 56 40 PM" src="https://user-images.githubusercontent.com/28743286/88001521-324cb280-cab5-11ea-926c-7589819a7963.png">

When user selects an installment, specifies Paused Reason and clicks [Save]:
<img width="527" alt="Screen Shot 2020-07-20 at 5 57 13 PM" src="https://user-images.githubusercontent.com/28743286/88001530-38db2a00-cab5-11ea-90c5-36c8c631ef57.png">

When user deselects all skipped installments and saves, the pause will be deactivated:
<img width="540" alt="Screen Shot 2020-07-20 at 6 15 33 PM" src="https://user-images.githubusercontent.com/28743286/88001545-41cbfb80-cab5-11ea-8235-c4d3af7bb55e.png">

Page displayed when the RD is closed preventing pause setup:
<img width="698" alt="Screen Shot 2020-07-20 at 6 19 26 PM" src="https://user-images.githubusercontent.com/28743286/88001702-9e2f1b00-cab5-11ea-89a0-1d32670166cd.png">

Here is the screenshot displaying the error message when a user does not have permission on the Apex class. Note this is not a label but wanted to let you know how the page will look like.
<img width="637" alt="Screen Shot 2020-07-20 at 5 52 37 PM" src="https://user-images.githubusercontent.com/28743286/88001613-6aec8c00-cab5-11ea-881c-47e9d3d94d01.png">